doc/backend.md

```diff
@@ -23,18 +23,18 @@ DeePMD-kit does not use the TensorFlow v2 API but uses the TensorFlow v1 API (`t
 [PyTorch](https://pytorch.org/) 2.0 or above is required.
 While `.pth` and `.pt` are the same in the PyTorch package, they have different meanings in the DeePMD-kit to distinguish the model and the checkpoint.
 
-### DPModel {{ dpmodel_icon }}
+### DP {{ dpmodel_icon }}
```

Comment:
Other places also need to update, for example https://deepmodeling--3385.org.readthedocs.build/projects/deepmd/en/3385/model/train-se-e2-a.html
Please do a global search in the doc directory.